doc/Caliptra.md

```diff
@@ -931,22 +938,86 @@ A detailed description of the POST and CAST KATs can be found at csrc.nist.gov.
+| Owner ECDSA Public Key | 96 | ECDSA P384 public key used to verify the Firmware Manifest Header Signature. <br> **X-Coordinate:** Public Key X-Coordinate (48 bytes) <br> **Y-Coordinate:** Public Key Y-Coordinate (48 bytes)|
+| Owner LMS Public Key | 48 | LMS public key used to verify the Firmware Manifest Header Signature. <br> **tree_type:** LMS Algorithm Type (4 bytes) <br> **otstype:** LMS Ots Algorithm Type (4 bytes) <br> **id:**  (16 bytes) <br> **digest:**  (24 bytes) |
+| Owner ECDSA Signature | 96 | Vendor ECDSA P384 signature of the Firmware Manifest header hashed using SHA384. <br> **R-Coordinate:** Random Point (48 bytes) <br> **S-Coordinate:** Proof (48 bytes) |
+| Owner LMS Signature | 1620 | Owner LMS signature of the Firmware Manifest header hashed using SHA384. <br> **q:** Leaf of the Merkle tree where the OTS public key appears (4 bytes) <br> **ots:** Lmots Signature (1252 bytes) <br> **tree_type:** Lms Algorithm Type (4 bytes) <br> **tree_path:** Path through the tree from the leaf associated with the LM-OTS signature to the root. (360 bytes) |
+| Reserved | 8 | Reserved 8 bytes |
+<br>
+
+*Table 17: Firmware manifest header*
+| Field | Size (bytes) | Description|
+|-------|--------|------------|
+| Revision | 8 | 8-byte version of the firmware image bundle |
+| Vendor ECDSA public key index | 4 | The hint to ROM to indicate which ECDSA public key it should first use. |
+| Vendor LMS public key index | 4 | The hint to ROM to indicate which LMS public key it should first use. |
+| Flags | 4 | Feature flags. <br> **Bit0:** - Interpret the pl0_pauser field. If not set, all PAUSERs are PL1 <br>**Bit1-Bit31:** Reserved |
+| TOC Entry Count | 4 | Number of entries in TOC. |
+| PL0 PAUSER | 4 | The PAUSER with PL0 privileges. |
+| TOC Digest | 48 | SHA384 Digest of table of contents. |
+| Vendor Data | 40 | Vendor Data. <br> **Not Before:** Vendor Start Date [ASN1 Time Format] for Caliptra-issued certificates (15 bytes) <br> **Not After:** Vendor End Date [ASN1 Time Format] for Caliptra-issued certificates (15 bytes) <br> **Reserved:** (10 bytes) |
+| Owner Data | 40 | Owner Data. <br> **Not Before:** Owner Start Date [ASN1 Time Format] for Caliptra-issued certificate. Takes preference over vendor start date (15 bytes) <br> **Not After:** Owner End Date [ASN1 Time Format] for Caliptra-issued certificates. Takes preference over vendor end date (15 bytes) <br> **Reserved:** (10 bytes) |
+
+*Table 18: Table of contents*
+| Field | Size (bytes) | Description|
+|-------|--------|------------|
+| TOC Entry Id | 4 | TOC Entry ID. The fields can have following values: <br> **0x0000_0001:** FMC  <br> **0x0000_0002:** Runtime |
+| Image Type | 4 | Image Type that defines format of the image section <br> **0x0000_0001:** Executable |
```

Comment:
`Image Type that defines format of the image section` -> `Image Type that defines the format of the image section`